<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GenericCommandDialog</class>
 <widget class="QDialog" name="GenericCommandDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>638</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Command Page</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>30</y>
     <width>91</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Close</set>
   </property>
  </widget>
  <widget class="QLCDNumber" name="packetId">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>30</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="packetIdLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>10</y>
     <width>66</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Packet ID</string>
   </property>
  </widget>
  <widget class="QLabel" name="subSystemCommandPageLabel">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Subsystem</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="subSystemTextBrowser">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>221</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>80</y>
     <width>81</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Command</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>10</y>
     <width>61</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Send To:</string>
   </property>
  </widget>
  <widget class="QTextBrowser" name="commandAddressTextBrowser">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>30</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>531</width>
     <height>481</height>
    </rect>
   </property>
   <property name="verticalScrollBarPolicy">
    <enum>Qt::ScrollBarAlwaysOn</enum>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>511</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>0</width>
      <height>1000</height>
     </size>
    </property>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>530</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>490</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>450</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>410</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>290</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>570</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>610</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>650</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>690</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>730</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QTextBrowser" name="itemLabelTextBrowser_20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>770</y>
       <width>301</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_1">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_2">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>50</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_3">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>90</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_4">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>130</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_5">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>170</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_6">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>210</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_7">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>250</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_8">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>290</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_9">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>330</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_10">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>370</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_11">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>410</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_12">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>450</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_13">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>490</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_14">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>530</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_15">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>570</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_16">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>610</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_17">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>650</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_18">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>690</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_19">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>730</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="paramsButton_20">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>770</y>
       <width>61</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Params</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_1">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>10</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_2">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>50</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_3">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>90</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_4">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>130</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_5">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>170</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_6">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>210</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_7">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>250</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_8">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>290</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_9">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>330</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_10">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>370</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_11">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>410</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_12">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>450</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_13">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>490</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_14">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>530</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_15">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>570</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_16">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>610</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_17">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>650</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_18">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>690</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_19">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>730</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SendButton_20">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>770</y>
       <width>71</width>
       <height>27</height>
      </rect>
     </property>
     <property name="text">
      <string>Send</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>GenericCommandDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GenericCommandDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
